pulp_ansible/app/galaxy/v3/filters.py

```diff
@@ -64,7 +83,6 @@ class CollectionVersionSearchFilter(FilterSet):
             "collection_version__pulp_created": "pulp_created",
             "collection_version__namespace": "namespace",
             "collection_version__name": "name",
-            "collection_version__version": "version",
```

Comment:
What is this change doing?

Reply:
Well, since we have a custom `SemanticVersionOrderingFilter`, we don't want this wrong sorting by collection version.